runtime/sema/crypto_algorithm_types.go

```diff
@@ -213,3 +221,8 @@ SHA3_256 is Secure Hashing Algorithm 3 (SHA-3) with a 256-bit digest
 const HashAlgorithmDocStringSHA3_384 = `
 SHA3_384 is Secure Hashing Algorithm 3 (SHA-3) with a 384-bit digest
 `
+
+const HashAlgorithmDocStringKMAC_128_BLS_BLS12381 = `
+KMAC_128_BLS_BLS12381 is a variant of KMAC_128 hashing algorithm, that
```

Comment:
Same here:

KMAC128_BLS_BLS12381 is an instance of KMAC128 mac algorithm, that can be used as the hashing algorithm for BLS signature scheme on the curve BLS12-381.